README.md

```diff
@@ -746,6 +747,32 @@ All Kubernetes resources that are currently supported by the Kubernetes provider
 * Because Terraform flatmap uses "." to detect the keys for unflattening the maps, some keys with "." in their names are being considered as the maps.
 * Since the library assumes empty strings to be empty values (not "0"), there are some issues with optional integer keys that are restricted to be positive.
 
+### Use with RabbitMQ
+
+Example:
+
+```
+ terraformer import rabbitmq --resources=vhosts,queues,exchanges
```

Comment:
Can you add example with username, password and host?

Reply:
Done.